src/stakingSetup.js

```diff
@@ -0,0 +1,153 @@
+const Keyring = require('@polkadot/keyring').default;
+const testingPairs = require('@plugnet/keyring/testingPairs');
+let log = require('console-log-level')({ level: 'info' });
+const DEFAULT_SLEEP = 5000;
+const keyring = new Keyring({ type: 'sr25519' });
+const defaulKeyrings = testingPairs.default({ type: 'sr25519' });
+
+
+const controllersKeyring = [
+    defaulKeyrings.bob,
+    defaulKeyrings.charlie,
+    defaulKeyrings.eve
+]; 
+
+const controllerAddrs = [
+    "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty", // BOB
+    "5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y", // CHARLIE
+    "5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw" // EVE
+]
+
+async function createStashAccounts(api) {
+    stashAccount = []
+    stashNames = ["BOB_STASH", "CHARLIE_STASH", "EVE_STASH"];
+    acliceKeyring = defaulKeyrings.alice;
+
+    for (i = 0; i < stashNames.length; i++) {
+        name = stashNames[i];
+        stashAccount.push(keyring.addFromUri(name, { name: name }));
+    }
+
+    for (account of stashAccount) {
+        log.info("SETTING UP FOR CENNZ -------------- \n Accout raw seed: " + account.meta.name)
+        log.info("Transfering cennz to " + account.meta.name)            
+            const cennz = await api.tx.genericAsset.transfer(16000, account.address, "100_000_000_000_000")
+            .signAndSend(acliceKeyring, (result) => {
+                log.info(`Current status is ${result.status}`);
+
+                if (result.isCompleted) {
+                    if (result.isFinalized) {
+                        hash = result.status.asFinalized;
+                    }
+                    else { // error
+                        log.info("Transaction Rejected");
+                        throw [APP_FAIL_TRANSACTION_REJECTED, `Transaction rejected ${result.status} for ${account.meta.name}`];
+                    }
+                    cennz();
+                }
+            })
+            .catch(err => transaction_error = err);
+            await sleep();
+            
+    }
+
+    for (account of stashAccount) {
```

Comment:
This codeblock repeats the previous one with only one change - I suggest putting into a function where you can set the name and asset_id.
Otherwise any improvement you make to one code might not be made to the other (there are already some examples, such as the throw string which show this)